### PR TITLE
Feature/bring back 96khz mode

### PR DESCRIPTION
**Pull request by damosvil**
This commit brings back 96khz mode because there seem to be lots of people using it.
It also fixes broken braking values in previous dynamic pwm pr

### CODE REVIEW

File: src/Bluejay.asm [hunk 1]
```diff
@@ -374,6 +378,7 @@ Eep_Pgm_Brake_On_Stop: DB DEFAULT_PGM_BRAKE_ON_STOP ; EEPROM copy of programmed
 Eep_Pgm_LED_Control: DB DEFAULT_PGM_LED_CONTROL ; EEPROM copy of programmed LED control
 Eep_Pgm_Power_Rating: DB DEFAULT_PGM_POWER_RATING ; EEPROM copy of programmed power rating
 Eep_Pgm_Safety_Arm: DB DEFAULT_PGM_SAFETY_ARM ; Various flag settings: bit 0 is require edt enable to arm
+Eep_Pgm_96to48_Threshold: DB DEFAULT_48to24_THRESHOLD ; Threshold to move between 96 and 48 khz PWM frequency
```
Comment (stylesuxx) [inline src/Bluejay.asm:381]: Wrong default value, should be `DEFAULT_96to48_THRESHOLD`

File: src/Modules/Isrs.asm [hunk 2]
```diff
@@ -408,7 +418,31 @@ t1_int_run_48khz:
...
+    cpl  A
+    anl  A, #1
+    mov  Temp3, A
+
+    ; Set PCA to work at 96khz
+    mov  PCA0PWM, #81h
+
 ELSE
+
+    ; Center aligned PWM
```
Comment (stylesuxx) [inline src/Modules/Isrs.asm:445]: Indentation.

**Review by stylesuxx — CHANGES_REQUESTED** [review]

File: src/Bluejay.asm (hunk 1 above)
Comment (damosvil) [reply src/Bluejay.asm]: Fixed

File: src/Modules/Isrs.asm (hunk 2 above)
Comment (damosvil) [reply src/Modules/Isrs.asm]: Replaced comment by small banner

File: src/Modules/Isrs.asm [hunk 3]
```diff
@@ -370,12 +370,24 @@ t1_int_zero_rcp_checked_check_limit:
...
+
+    ; Choose between 48khz and 24khz
+    clr  C
     subb A, Throttle_48to24_Threshold
     jc   t1_int_run_48khz
 
+
 IF PWM_CENTERED == 0
+
+; *******************
```
Comment (stylesuxx) [inline src/Modules/Isrs.asm:388]: I don't want to be a pain in the ass, but maybe we could keep the banners consistent?

```asm
;**** **** **** **** **** **** **** **** **** **** **** **** ****
; Edge aligned PWM
;**** **** **** **** **** **** **** **** **** **** **** **** ****
```

Comment (damosvil) [reply src/Modules/Isrs.asm]: Fixed

**Comment by damosvil** [comment]
To be merged after:
https://github.com/bird-sanctuary/bluejay/pull/172

@stylesuxx It requires changes in configurator to add 96khz threshold. Please, when possible could you add them?

**Comment by stylesuxx** [comment]
Will do, I should be able to get it done either today, latest tomorrow.

**Comment by stylesuxx** [comment]
Merge after https://github.com/bird-sanctuary/bluejay/pull/183

**Review by hyp0dermik-code — APPROVED** [review]

**Review by stylesuxx — APPROVED** [review]

**Pull request merged**